perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -996,15 +1005,162 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     .carousselParent, .carousselImage, .v-carousel__item, .v-window__container {
        max-height: 250px !important;
     }
+
+    .product-list {
+      .toolbarProduct {
+        padding: 15px 5px 15px 15px;
```

Comment:
You can use pa-4 directly (https://vuetifyjs.com/en/styles/spacing/#how-it-works)